autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -516,6 +670,31 @@ def file_output(
             Y_valid_pred: np.ndarray,
             Y_test_pred: np.ndarray
     ) -> Tuple[Optional[float], Dict]:
+        """
+        This method decides what file outputs are written to disk.
```

Comment:
which file to write the outputs

Reply:
Thanks, I have improved the docstring with this!